examples/gno.land/p/demo/json/decode_test.gno

```diff
@@ -0,0 +1,567 @@
+package json
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+)
+
+type testNode struct {
+	name  string
+	input []byte
+	_type ValueType
+	value []byte
+}
+
+func simpleValid(test *testNode, t *testing.T) {
```

Comment:
Just a note for all helper methods here, `testing.T` should always be the first param

Reply:
oh I mistake. thanks!